### PR TITLE
Release highlights 2021.11

### PR DESCRIPTION
**Pull request by mcarrano**
Adds release highlights for 2021.11 release.

### CODE REVIEW

File: release-highlights/release-highlights-2021.11/release-highlights.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### [New horizontal sub-navigation](https://www.patternfly.org/v4/components/navigation#horizontal-subnav)
+
+![page with vertical and horizontal sub-navigation](./img/horizontal-sub-nav.png)
+
+Introduces a new horizontal secondary navigation bar that will replace the legacy tertiary navigation to use for exposing local navigation within a page. The new navigation bar is better differentiated from horizontal tabs. See a [full page demo](https://www.patternfly.org/v4/components/navigation/react-demos#horizontal-subnav).
+
+### [Multi-select rows in a table](https://www.patternfly.org/v4/components/table#composable-selectable)
+Users can now use `shift+select` to select a range of rows in composable table component.
+
+### [Popover diagonal positioning](https://www.patternfly.org/v4/components/popover)
```
Comment (evwilkin) [inline release-highlights/release-highlights-2021.11/release-highlights.md:27]: Other components link to the specific example showing the update, can we change this to 

https://www.patternfly.org/v4/components/popover/#advanced

**Review by evwilkin — CHANGES_REQUESTED** [review]
Just request that the Popover link goes to the Advanced example showing the updates

File: release-highlights/release-highlights-2021.11/release-highlights.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* [Auto-complete search input](https://github.com/patternfly/patternfly-react/issues/6210) - Adds auto-complete behavior to the search input component to expose a list of suggestions as the user types.
+
+* [Refactor page to use new masthead](https://github.com/patternfly/patternfly-react/issues/6174) - refactoring our page demos to utilize the new masthead component. This will provide a more flexible and responsive [masthead](https://www.patternfly.org/v4/components/masthead) for future projects.
+
+
+### 2021.13 (October 8)
+
+* [Horizontal table scrolling with sticky columns](https://github.com/patternfly/patternfly/issues/4218) - Allow a table to scroll horizontally while keeping some number of columns always visible.
+
+* [Progress stepper component](https://github.com/patternfly/patternfly/issues/4214) - introduce a new component that tracks progress of a linear, step-wise task.
```
Comment (nicolethoen) [inline release-highlights/release-highlights-2021.11/release-highlights.md:57]: Use consistent sentence case and verb tense. Some of the coming soon uses `Add` and others say `Adds`.

File: release-highlights/release-highlights-2021.11/release-highlights.md (hunk 1 above)
Comment (mcarrano) [reply release-highlights/release-highlights-2021.11/release-highlights.md]: That makes sense

File: release-highlights/release-highlights-2021.11/release-highlights.md (hunk 2 above)
Comment (mcarrano) [reply release-highlights/release-highlights-2021.11/release-highlights.md]: Good catch. I've made these all "Add" or the equivalent.

**Comment by mcarrano** [comment]
@evwilkin @nicolethoen I addressed your feedback. Please review again and merge when ready.

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Pull request merged**